src/deepsparse/loggers/function_logger.py

```diff
@@ -31,49 +33,70 @@ class FunctionLogger(BaseLogger):
     according to FunctionLogger's attributes
 
     :param logger: A child DeepSparse Logger object
-    :param target_identifier: The string that needs to match the
-        `identifier` (argument to the log() method),
-        so that the FunctionLogger applies
-        the metric function and logs the data
-    :param function_name: Name of the metric function
-    :param function: The metric function to be applied
-    :param frequency: The frequency with which the metric
-        name is to be applied
+    :param target_logging_configs: A list of TargetLoggingConfig objects
     """
 
     def __init__(
         self,
         logger: BaseLogger,
-        target_identifier: str,
-        function: Callable[[Any], Any],
-        function_name: str = None,
-        frequency: int = 1,
+        target_logging_configs: List["TargetLoggingConfig"],  # noqa F821
```

Comment:
Was function logger in previous form not easy to use?

Reply:
We are still in the process of figuring out the tradeoff between having fewer amount of loggers instantiated overall and less coupling between the FunctionLogger and the server inputs.